themes/finna2/templates/organisationinfo/elements/location-quick-info.phtml

```diff
@@ -0,0 +1,96 @@
+<!-- START of: finna - organisationinfo/elements/location-quick-info.phtml -->
+<?php
+  $renderInterval = function (int $opens, int $closes, bool $minutes): string {
+    return $this->render('organisationinfo/elements/helpers/time-interval.phtml', compact('opens', 'closes', 'minutes'));
+  };
+?>
+<div class="location-basic-info">
+  <h3 class="location-title"><?=$this->escapeHtml($orgInfo['name'])?></h3>
```

Comment:
Otsikoon aria-atomic="true" aria-live="polite" ja otsikon sisälle <span class="sr-only"><?=$this->transEsc('organisation_info_selected_service_point')?></span>

Reply:
Lisätty.